docs/user_guide.md

```diff
@@ -252,6 +256,95 @@ The images can be filtered by labels:
+
+System-wide CVE whitelists apply to all of the projects in a Harbor instance.
+
+1. Go to **Configuration** > **System Settings**.
+1. Under **Deployment security**, click **Add**. 
+   ![System-wide CVE whitelist](img/cve-whitelist1.png)
+1. Enter the list of CVE IDs to ignore during vulnerability scanning. 
+   ![Add system CVE whitelist](img/cve-whitelist2.png)
+
+   Either use a comma-separated list or newlines to add multiple CVE IDs to the list.
+1. Click **Add** at the bottom of the window to add the list.
+1. Optionally uncheck the **Never expires** checkbox and use the calendar selector to set an expiry date for the whitelist.
+   ![Add system CVEs](img/cve-whitelist3.png)
+1. Click **Save** at the bottom of the page to save your settings.
+   
+After you have created a system whitelist, you can remove CVE IDs from the list by clicking the delete button next to it in the list. You can click **Add** to add more CVE IDs to the system whitelist.
+
+![Add and remove system CVEs](img/cve-whitelist4.png)
+
+### Configure a Per-Project CVE Whitelist
+
+By default, the system whitelist is applied to all projects. You can configure different CVE whitelists for individual projects, that override the system whitelist. Per-project CVE whitelists only apply to the projects in which you create them. 
+
+1. Go to **Projects**, select a project, and select **Configuration**.
+1. Under **CVE whitelist**, select **Project whitelist**. 
+   ![Project CVE whitelist](img/cve-whitelist5.png)
+1. Optionally click **Add System** to add all of the CVE IDs from the system CVE whitelist to this project whitelist.
+1. Click **Add** and enter a list of additional CVE IDs to ignore during vulnerability scanning of this project. 
+   ![Add project CVEs](img/cve-whitelist6.png)
+
+   Either use a comma-separated list or newlines to add multiple CVE IDs to the list.
+1. Click **Add** at the bottom of the window to add the CVEs to the project whitelist.
+1. Optionally uncheck the **Never expires** checkbox and use the calendar selector to set an expiry date for the whitelist.
+1. Click **Save** at the bottom of the page to save your settings.
+
+After you have created a project whitelist, you can remove CVE IDs from the list by clicking the delete button next to it in the list. You can click **Add** at any time to add more CVE IDs to the whitelist for this project. 
+
+If CVEs are added to the system whitelist after you have created a project whitelist, click **Add System** to add the new entries from the system whitelist to the project whitelist. 
+
+**NOTE**: If CVEs are deleted from the system whitelist after you have created a project whitelist, and if you added the system whitelist to the project whitelist, you must manually remove the deleted CVEs from the project whitelist. If you click **Add System** after CVEs have been deleted from the system whitelist, the deleted CVEs are not automatically removed from the project whitelist.

```

Comment:
regarding 'If CVEs are deleted from the system whitelist after you have created a project whitelist, and if you added the system whitelist to the project whitelist, you must manually remove the deleted CVEs from the project whitelist.'  Should this be the correct behavior?  I wonder if these should be automatically removed from the project, ie 'de-merged' from the project whitelist at runtime.  Or is that not possible?  @reasonerjt

Reply:
@xaleeks and @reasonerjt I tested as follows:

- Created a per-project list
- Created a system-wide list
- In the project list, clicked Add System and saw the CVEs from the system list appear in the project list.
- Deleted a CVE from the system list.
- Went back to the project list and clicked Add System again to see if that updated the project list, but the CVE that I removed from the system list remained in the project list

Reply:
Thx for verifying.  I'll make a note on this.  Doc-wise, this is fine, lgtm